docs/source/conf.py

```diff
@@ -59,6 +59,26 @@
     "beta_status",
 ]
 
+# We override sphinx-gallery's example header to prevent sphinx-gallery from
+# creating a note at the top of the renderred notebook.
+# https://github.com/sphinx-gallery/sphinx-gallery/blob/451ccba1007cc523f39cbcc960ebc21ca39f7b75/sphinx_gallery/gen_rst.py#L1267-L1271
+# This is because we also want to add a link to google collab, so we write our own note in each example.
```

Comment:
quick explanation

sphinx-gallery adds this note by default on top of each example:

![image](https://github.com/pytorch/vision/assets/1190450/13cfb35a-f156-4030-b1ff-cb3d3d4e4bbe)


We want to add links to collab, and I want to avoid having 2 notes. So I removed the note automatically added by sphinx-gallery and manually added it on each example, along with the "try on collab" link.

It's a bit more work but I think it's worth it. I'll open an issue on the sphinx-gallery repo to see if there's a clean way to hook into that automatically-generated note.